app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i.freeze
+  validates :email, uniqueness: true,
+   length: {maximum: Settings["user"]["length"]},
+   format: {with: VALID_EMAIL_REGEX}
+  validates :password, length:
+   {in: Range.new(Settings["user"]["pwd_min"], Settings["user"]["pwd_max"])}
```

Comment:
```suggestion
   {in: Range.new(Settings.user.pwd_min, Settings.user.pwd_max)}
```
Viết thế này cho dễ nhìn em ơi